### ISSUE TITLE
Release Che 7.34.2

### ISSUE THREAD

**Issue by nickboldt** [eclipse-che/che#20284]
### List of pending issues / PRs
* [ ] description #xxx https://github.com/eclipse/che/issues/xxx

### Release status

The following will be released via [che-release](https://github.com/eclipse/che-release/blob/master/cico_release.sh) according to a series of phases:

1. [x] che-machine-exec, che-theia, che-devfile-registry, che-dashboard, devworkspace-operator, jwtproxy, kubernetes image puller
2. [x] che-plugin-registry (once che-theia and machine-exec are done)
3. [x] che-parent and che server
4. [x] devworkspace-che-operator
5. [x] che-operator

Each phase will [send a Mattermost notification to the Eclipse Che releases channel](https://mattermost.eclipse.org/eclipse/channels/eclipse-che-releases).

Then, these steps will be done, once the above projects are released and PRs are merged:

- [ ] [chectl](https://github.com/che-incubator/chectl/actions/workflows/release-stable-PRs.yml) _(depends on che-operator)_
- [ ] [Che community operator PRs](https://github.com/operator-framework/community-operators/pulls?q=%22Update+eclipse-che+operator%22+is%3Aopen) _(depends on che-operator)_
- [ ] [che-docs PR](https://github.com/eclipse/che-docs/pulls/che-bot) _(depends on che-operator)_

If this is a .0 release:

- [ ] complete current milestone
  - [ ] move incomplete *deferred* issues to backlog
  - [ ] move incomplete *WIP* issues to next milestone
  - [ ] close completed issues
  - [ ] close milestone

| Process <sup>[1]</sup> | Script | Action | Container(s) + Artifact(s) |
| --- | --- | --- | --- |
| [che-release](https://github.com/eclipse/che-release/blob/master/RELEASE.md) | [cico_release.sh](https://github.com/eclipse/che-release/blob/master/cico_release.sh) | [Action](https://github.com/eclipse/che-release/actions?query=workflow%3A%22Release+-+Orchestrate+Overall+Release+Phases%22) | n/a |
| [che-theia](https://github.com/eclipse/che-theia/blob/main/RELEASE.md) | [make-release.sh](https://github.com/eclipse/che-theia/blob/main/make-release.sh) | [Action](https://github.com/eclipse/che-theia/actions?query=workflow%3A%22Release+Che+Theia%22) | [`eclipse/che-theia`](https://quay.io/eclipse/che-theia) |
| [che-machine-exec](https://github.com/eclipse-che/che-machine-exec/blob/master/RELEASE.md) | [make-release.sh](https://github.com/eclipse-che/che-machine-exec/blob/master/make-release.sh) | [Action](https://github.com/eclipse-che/che-machine-exec/actions?query=workflow%3A%22Release+Che+Machine+Exec%22) | [`eclipse/che-machine-exec`](https://quay.io/eclipse/che-machine-exec)|
| [che-devfile-registry](https://github.com/eclipse/che-devfile-registry/blob/master/RELEASE.md) | [make-release.sh](https://github.com/eclipse/che-devfile-registry/blob/master/make-release.sh) | [Action](https://github.com/eclipse/che-devfile-registry/actions?query=workflow%3A%22Release+Che+Devfile+Registry%22) | [`eclipse/che-devfile-registry`](https://quay.io/eclipse/che-devfile-registry)|
| [che-plugin-registry](https://github.com/eclipse/che-plugin-registry/blob/master/RELEASE.md) | [make-release.sh](https://github.com/eclipse/che-plugin-registry/blob/master/make-release.sh) | [Action](https://github.com/eclipse/che-plugin-registry/actions?query=workflow%3A%22Release+Che+Plugin+Registry%22) | [`eclipse/che-plugin-registry`](https://quay.io/eclipse/che-plugin-registry)|
| [che-parent](https://github.com/eclipse/che-parent/blob/master/RELEASE.md) | [make-release.sh](https://github.com/eclipse/che-parent/blob/master/make-release.sh) | [Action](https://github.com/eclipse/che/actions?query=workflow%3A%22Release+Che+Server%22) | [che-server](https://search.maven.org/search?q=a:che-server) <sup>[2]</sup> |
| [che-dashboard](https://github.com/eclipse-che/che-dashboard/blob/main/RELEASE.md) | [make-release.sh](https://github.com/eclipse-che/che-dashboard/blob/master/make-release.sh) | [Action](https://github.com/eclipse-che/che-dashboard/actions?query=workflow%3A%22Release+Che+Dashboard%22) | [`che-dashboard`](https://quay.io/repository/eclipse/che-dashboard?tag=next&tab=tags) |
| [che](https://github.com/eclipse-che/che-server/blob/HEAD/RELEASE.md) | [make-release.sh](https://github.com/eclipse-che/che-server/blob/HEAD/make-release.sh) | [Action](https://github.com/eclipse-che/che-server/actions?query=workflow%3A%22Release+Che+Server%22) | [`eclipse/che-server`](https://quay.io/eclipse/che-server),<br/>[`eclipse/che-endpoint-watcher`](https://quay.io/eclipse/che-endpoint-watcher),<br/> [`eclipse/che-keycloak`](https://quay.io/eclipse/che-keycloak),<br/> [`eclipse/che-postgres`](https://quay.io/eclipse/che-postgres),<br/> [`eclipse/che-server`](https://quay.io/eclipse/che-server),<br/> [`eclipse/che-e2e`](https://quay.io/eclipse/che-e2e) |
| [devworkspace-operator (controller)](https://github.com/devfile/devworkspace-operator/) | [make-release.sh](https://github.com/devfile/devworkspace-operator/blob/main/make-release.sh) | [Action](https://github.com/devfile/devworkspace-operator/actions/workflows/release.yml) | [`devfile/devworkspace-controller`](https://quay.io/repository/devfile/devworkspace-controller?tab=tags)|
| [devworkspace-che-operator](https://github.com/che-incubator/devworkspace-che-operator/) | [make-release.sh](https://github.com/che-incubator/devworkspace-che-operator/blob/main/make-release.sh) | [Action](https://github.com/che-incubator/devworkspace-che-operator/actions/workflows/release.yml) | [`che-incubator/devworkspace-che-operator`](https://quay.io/repository/che-incubator/devworkspace-che-operator?tab=tags)|
| [che-operator](https://github.com/eclipse-che/che-operator/blob/master/RELEASE.md) | [make-release.sh](https://github.com/eclipse-che/che-operator/blob/master/make-release.sh) | [Action](https://github.com/eclipse-che/che-operator/actions?query=workflow%3A%22Release+Che+Operator%22) | [`eclipse/che-operator`](https://quay.io/eclipse/che-operator)|
| [chectl](https://github.com/che-incubator/chectl/blob/master/RELEASE.md) | [make-release.sh](https://github.com/che-incubator/chectl/blob/master/make-release.sh) | [Action](https://github.com/che-incubator/chectl/actions) | [chectl releases](https://github.com/che-incubator/chectl/releases)

<sup>[1]</sup> Overall process owner: @mkuznyetsov

**Comment by nickboldt** [comment]
had to remove dwo from the release process... https://github.com/eclipse-che/che-release/pull/44
now running in https://github.com/eclipse-che/che-release/actions/runs/1120366466

**Comment by nickboldt** [comment]
respinning plugin reg for the npmjs fix https://github.com/eclipse-che/che-plugin-registry/runs/3302950545?check_suite_focus=true
* died because nothing new to commit (existing release; need to add an `|| true` if nothing to push)

meanwhile operator push hasn't started ?

**Comment by nickboldt** [comment]
Operator release failed with:
```
2021-08-12T12:25:04.8397086Z Switched to branch '7.34.x'
2021-08-12T12:25:04.8398487Z Your branch is up to date with 'origin/7.34.x'.
2021-08-12T12:25:04.8590354Z Switched to branch '7.34.2-release'
2021-08-12T12:25:05.5281150Z [INFO] Release '7.34.2' from branch '7.34.x'
2021-08-12T12:25:05.5282374Z [INFO] Push git changes into 7.34.2-release branch
2021-08-12T12:25:06.7024224Z remote: 
2021-08-12T12:25:06.7026117Z remote: Create a pull request for '7.34.2-release' on GitHub by visiting:        
2021-08-12T12:25:06.7027654Z remote:      https://github.com/eclipse-che/che-operator/pull/new/7.34.2-release        
2021-08-12T12:25:06.7028646Z remote: 
2021-08-12T12:25:06.7099795Z To https://github.com/eclipse-che/che-operator
2021-08-12T12:25:06.7101158Z  * [new branch]        7.34.2-release -> 7.34.2-release
2021-08-12T12:25:07.6991085Z To https://github.com/eclipse-che/che-operator
2021-08-12T12:25:07.6992552Z  * [new tag]           7.34.2 -> 7.34.2
2021-08-12T12:25:07.7108279Z [INFO] createPRToXBranch :: Create pull request into 7.34.x branch
2021-08-12T12:25:08.8411156Z https://github.com/eclipse-che/che-operator/pull/1008
2021-08-12T12:25:08.8424133Z [INFO] createPRToMainBranch :: Create pull request into main branch to copy csv
2021-08-12T12:25:08.8425040Z [INFO] Reset changes in main branch
2021-08-12T12:25:08.8634703Z HEAD is now at 13a53ac0 ci: Release OLM files to 7.34.2
2021-08-12T12:25:08.8960651Z Switched to branch 'main'
2021-08-12T12:25:08.8961552Z Your branch is up to date with 'origin/main'.
2021-08-12T12:25:09.1262216Z From https://github.com/eclipse-che/che-operator
2021-08-12T12:25:09.1263473Z  * branch              HEAD       -> FETCH_HEAD
2021-08-12T12:25:09.3411444Z From https://github.com/eclipse-che/che-operator
2021-08-12T12:25:09.3413409Z  * branch              main       -> FETCH_HEAD
2021-08-12T12:25:09.3534913Z Already up to date.
2021-08-12T12:25:09.3775972Z Switched to a new branch 'copy-csv-to-main'
2021-08-12T12:25:09.3970933Z Applied patch to 'bundle/stable-all-namespaces/eclipse-che-preview-openshift/manifests/che-operator.clusterserviceversion.yaml' cleanly.
2021-08-12T12:25:09.4015964Z Applied patch to 'bundle/stable-all-namespaces/eclipse-che-preview-openshift/manifests/che-operator.clusterserviceversion.yaml.diff' cleanly.
2021-08-12T12:25:09.4022931Z Applied patch to 'bundle/stable-all-namespaces/eclipse-che-preview-openshift/manifests/org.eclipse.che_chebackupserverconfigurations_crd.yaml.diff' cleanly.
2021-08-12T12:25:09.4028312Z Applied patch to 'bundle/stable-all-namespaces/eclipse-che-preview-openshift/manifests/org.eclipse.che_checlusterbackups_crd.yaml.diff' cleanly.
2021-08-12T12:25:09.4033285Z Applied patch to 'bundle/stable-all-namespaces/eclipse-che-preview-openshift/manifests/org.eclipse.che_checlusterrestores_crd.yaml.diff' cleanly.
2021-08-12T12:25:09.4059402Z Applied patch to 'bundle/stable-all-namespaces/eclipse-che-preview-openshift/manifests/org_v1_che_crd.yaml.diff' cleanly.
2021-08-12T12:25:09.4095935Z Applied patch to 'bundle/stable/eclipse-che-preview-kubernetes/manifests/che-operator.clusterserviceversion.yaml' with conflicts.
2021-08-12T12:25:09.4124992Z Applied patch to 'bundle/stable/eclipse-che-preview-kubernetes/manifests/che-operator.clusterserviceversion.yaml.diff' cleanly.
2021-08-12T12:25:09.4130166Z Applied patch to 'bundle/stable/eclipse-che-preview-kubernetes/manifests/org.eclipse.che_chebackupserverconfigurations_crd.yaml.diff' cleanly.
2021-08-12T12:25:09.4134641Z Applied patch to 'bundle/stable/eclipse-che-preview-kubernetes/manifests/org.eclipse.che_checlusterbackups_crd.yaml.diff' cleanly.
2021-08-12T12:25:09.4139018Z Applied patch to 'bundle/stable/eclipse-che-preview-kubernetes/manifests/org.eclipse.che_checlusterrestores_crd.yaml.diff' cleanly.
2021-08-12T12:25:09.4158580Z Applied patch to 'bundle/stable/eclipse-che-preview-kubernetes/manifests/org_v1_che_crd.yaml.diff' cleanly.
2021-08-12T12:25:09.4189354Z Applied patch to 'bundle/stable/eclipse-che-preview-openshift/manifests/che-operator.clusterserviceversion.yaml' with conflicts.
2021-08-12T12:25:09.4213931Z Applied patch to 'bundle/stable/eclipse-che-preview-openshift/manifests/che-operator.clusterserviceversion.yaml.diff' cleanly.
2021-08-12T12:25:09.4219664Z Applied patch to 'bundle/stable/eclipse-che-preview-openshift/manifests/org.eclipse.che_chebackupserverconfigurations_crd.yaml.diff' cleanly.
2021-08-12T12:25:09.4224087Z Applied patch to 'bundle/stable/eclipse-che-preview-openshift/manifests/org.eclipse.che_checlusterbackups_crd.yaml.diff' cleanly.
2021-08-12T12:25:09.4228268Z Applied patch to 'bundle/stable/eclipse-che-preview-openshift/manifests/org.eclipse.che_checlusterrestores_crd.yaml.diff' cleanly.
2021-08-12T12:25:09.4245656Z Applied patch to 'bundle/stable/eclipse-che-preview-openshift/manifests/org_v1_che_crd.yaml.diff' cleanly.
2021-08-12T12:25:09.4248267Z Applied patch to 'version/version.go' cleanly.
2021-08-12T12:25:09.4269170Z U bundle/stable/eclipse-che-preview-kubernetes/manifests/che-operator.clusterserviceversion.yaml
2021-08-12T12:25:09.4271219Z U bundle/stable/eclipse-che-preview-openshift/manifests/che-operator.clusterserviceversion.yaml
2021-08-12T12:25:09.4317301Z ##[error]Process completed with exit code 1.
``` 
-- https://github.com/eclipse-che/che-operator/runs/3311161177?check_suite_focus=true

**Comment by nickboldt** [comment]
Looks like I've worked around the problem and we have a 7.34.2 release.

Here's the diff that couldn't be applied:

```
2021-08-13T13:57:20.4379532Z U bundle/stable/eclipse-che-preview-kubernetes/manifests/che-operator.clusterserviceversion.yaml
2021-08-13T13:57:20.4382774Z U bundle/stable/eclipse-che-preview-openshift/manifests/che-operator.clusterserviceversion.yaml
2021-08-13T13:57:20.4428532Z + git diff
2021-08-13T13:57:20.4575380Z diff --cc bundle/stable/eclipse-che-preview-kubernetes/manifests/che-operator.clusterserviceversion.yaml
2021-08-13T13:57:20.4579959Z + git status --porcelain
2021-08-13T13:57:20.4580847Z index 299749dc,bbbaa7fb..00000000
2021-08-13T13:57:20.4583490Z --- a/bundle/stable/eclipse-che-preview-kubernetes/manifests/che-operator.clusterserviceversion.yaml
2021-08-13T13:57:20.4586117Z +++ b/bundle/stable/eclipse-che-preview-kubernetes/manifests/che-operator.clusterserviceversion.yaml
2021-08-13T13:57:20.4587606Z @@@ -998,8 -1006,10 +998,13 @@@ spec
2021-08-13T13:57:20.4589160Z                           value: quay.io/eclipse/che--traefik@sha256:df90799aaca1ad6fb9e06d311140035d2a0c2295a4f8f508f6b55ee056bb677e
2021-08-13T13:57:20.4590800Z                         - name: RELATED_IMAGE_single_host_gateway_config_sidecar
2021-08-13T13:57:20.4592531Z                           value: quay.io/che-incubator/configbump@sha256:175ff2ba1bd74429de192c0a9facf39da5699c6da9f151bd461b3dc8624dd532
2021-08-13T13:57:20.4594122Z ++<<<<<<< ours
2021-08-13T13:57:20.4594617Z ++=======
2021-08-13T13:57:20.4595679Z +                       - name: RELATED_IMAGE_devworkspace_che_operator
2021-08-13T13:57:20.4598323Z +                         value: quay.io/che-incubator/devworkspace-che-operator@sha256:d0e89a3d0043ab54a6465ed3f61423302b7236bab4bdc6f6b61b4dd16a1310d4
2021-08-13T13:57:20.4600324Z ++>>>>>>> theirs
2021-08-13T13:57:20.4601567Z                         - name: RELATED_IMAGE_devworkspace_controller
2021-08-13T13:57:20.4604069Z -                         value: quay.io/devfile/devworkspace-controller@sha256:f17dad6df3f2f0f7b245e05677293bef1d35a17e0349002f9e47816de03c0cdd
2021-08-13T13:57:20.4607129Z +                         value: quay.io/devfile/devworkspace-controller@sha256:e7e3da9a91a1f170df3cdf0acd15ff49323ad1c3aad9f88134751ec393d234c3
2021-08-13T13:57:20.4609239Z                         - name: RELATED_IMAGE_internal_rest_backup_server
2021-08-13T13:57:20.4610957Z                           value: quay.io/eclipse/che-backup-server-rest@sha256:24983ce54474bbe87239f23773faeb59a5471a6a4b20e731518ad5738bfea6d7
2021-08-13T13:57:20.4612586Z                         - name: RELATED_IMAGE_gateway_authentication_sidecar
2021-08-13T13:57:20.4614713Z diff --cc bundle/stable/eclipse-che-preview-openshift/manifests/che-operator.clusterserviceversion.yaml
2021-08-13T13:57:20.4616231Z index 4a17cb22,1c122b48..00000000
2021-08-13T13:57:20.4618447Z --- a/bundle/stable/eclipse-che-preview-openshift/manifests/che-operator.clusterserviceversion.yaml
2021-08-13T13:57:20.4621136Z +++ b/bundle/stable/eclipse-che-preview-openshift/manifests/che-operator.clusterserviceversion.yaml
2021-08-13T13:57:20.4625496Z @@@ -1052,8 -1060,10 +1052,13 @@@ spec
2021-08-13T13:57:20.4626847Z                           value: quay.io/eclipse/che--traefik@sha256:df90799aaca1ad6fb9e06d311140035d2a0c2295a4f8f508f6b55ee056bb677e
2021-08-13T13:57:20.4628285Z                         - name: RELATED_IMAGE_single_host_gateway_config_sidecar
2021-08-13T13:57:20.4630513Z                           value: quay.io/che-incubator/configbump@sha256:175ff2ba1bd74429de192c0a9facf39da5699c6da9f151bd461b3dc8624dd532
2021-08-13T13:57:20.4631843Z ++<<<<<<< ours
2021-08-13T13:57:20.4632208Z ++=======
2021-08-13T13:57:20.4632979Z +                       - name: RELATED_IMAGE_devworkspace_che_operator
2021-08-13T13:57:20.4635034Z +                         value: quay.io/che-incubator/devworkspace-che-operator@sha256:d0e89a3d0043ab54a6465ed3f61423302b7236bab4bdc6f6b61b4dd16a1310d4
2021-08-13T13:57:20.4636397Z ++>>>>>>> theirs
2021-08-13T13:57:20.4638598Z                         - name: RELATED_IMAGE_devworkspace_controller
2021-08-13T13:57:20.4640596Z -                         value: quay.io/devfile/devworkspace-controller@sha256:f17dad6df3f2f0f7b245e05677293bef1d35a17e0349002f9e47816de03c0cdd
2021-08-13T13:57:20.4643551Z +                         value: quay.io/devfile/devworkspace-controller@sha256:e7e3da9a91a1f170df3cdf0acd15ff49323ad1c3aad9f88134751ec393d234c3
2021-08-13T13:57:20.4645152Z                         - name: RELATED_IMAGE_internal_rest_backup_server
2021-08-13T13:57:20.4646616Z                           value: quay.io/eclipse/che-backup-server-rest@sha256:24983ce54474bbe87239f23773faeb59a5471a6a4b20e731518ad5738bfea6d7
2021-08-13T13:57:20.4648226Z                         - name: RELATED_IMAGE_gateway_authentication_sidecar
```
-- https://pipelines.actions.githubusercontent.com/rDJDjck52LoLMEl2iDZefMlGEPZPq1wO5p381Kn5YQ1XDQkc3L/_apis/pipelines/1/runs/23429/signedlogcontent/3?urlExpires=2021-08-13T14%3A31%3A56.5332329Z&urlSigningMethod=HMACV1&urlSignature=zYv6gmylf1dX6vGEbhkCoR7819IurBXDwmfTTqyOBRQ%3D

@tolusha @mmorhun ^ is that something we need to fix in 7.35.0 (and/or in 7.34.2?) cc: @sleshchenko
